### PR TITLE
fix(explore comma): make that the comma can be added by removing it from token separators…

### PR DESCRIPTION
**Pull request by prosdev0107**
### SUMMARY
Cannot add comma to Number Formatting

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Before:
Comma doesn't show up, and the cursor jumps to the beginning of the line. Typing a period then erases everything.
It is because that the comma is used as the token separator in select component.
After:
I fixed this issue by removing the comma from the token separators.
Screenshot at Feb 24 07-56-43

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by geido — CHANGES_REQUESTED** [review]
Hello @prosdev0107 thanks for the contribution! However, I think we shouldn't take this path as the comma is a very common separator. I think we need to override the separators in those instances where it is required. As @michael-s-molina pointed out also, the decimal places separators also vary by locale. Let me know if you want to tackle this problem!

**Comment by prosdev0107** [comment]
Hi @geido .
Thank you for your review.

As a second way for solving the 'Cannot add comma to Number Formatting' in explore, I can ignore the token separators in case of single selector. And so I mean that the token separators can only work in case of multiple or tags mode.

In other case, thank you if you let me know the solution for this.

**Comment by rusackas** [comment]
If you do enable the AntD prop, it might be good to add a control to the interactive Storybook.

**Comment by prosdev0107** [comment]
@michael-s-molina @evans .

Resolved the review.

**Review by michael-s-molina — CHANGES_REQUESTED** [review]
You don't need to create another property in `SelectProps`, you just need to add `tokenSeparators` in `PickedSelectProps` as you can see in my previous comment. You also misspelled `tokenSeparators` (you wrote `tokenSepErators`)

**Comment by prosdev0107** [comment]
@michael-s-molina 

Resolved your comment.

File: superset-frontend/src/components/Select/Select.tsx [hunk 1]
```diff
@@ -266,35 +267,33 @@ const getQueryCacheKey = (value: string, page: number, pageSize: number) =>
...
+  options,
+  pageSize = DEFAULT_PAGE_SIZE,
+  placeholder = t('Select ...'),
+  showSearch = true,
+  sortComparator = defaultSortComparator,
+  tokenSeparators,
+  value,
+  ...props
+}: SelectProps,
+ref: RefObject<HTMLInputElement>,) => {
```
Comment (michael-s-molina) [inline superset-frontend/src/components/Select/Select.tsx:296]:
```suggestion
ref: RefObject<HTMLInputElement>) => {
```

File: superset-frontend/src/components/Select/Select.tsx [hunk 2]
```diff
@@ -299,13 +300,13 @@ const Select = (
     onError,
     onChange,
     onClear,
-    onDropdownVisibleChange,
```
Comment (diegomedina248) [inline superset-frontend/src/components/Select/Select.tsx:302]: This prop should stay or remove the code that uses it.

**Review by diegomedina248 — CHANGES_REQUESTED** [review]

File: superset-frontend/src/components/Select/Select.tsx (hunk 2 above)
Comment (michael-s-molina) [reply superset-frontend/src/components/Select/Select.tsx:302]: This property is used and should be restored.

Comment (prosdev0107) [reply superset-frontend/src/components/Select/Select.tsx]: @michael-s-molina 
Resolved

**Review by diegomedina248 — APPROVED** [review]

**Comment by rusackas** [comment]
@prosdev0107 just give this a quick rebase, and we should be good to go.

**Review by michael-s-molina — APPROVED** [review]
LGTM. Thanks for the fix.

**Review by geido — APPROVED** [review]
LGTM! @prosdev0107 you have a conflict to resolve. Thanks!

**Pull request merged**